### ISSUE TITLE
Bug: plex_update.ps1 issue?

### ISSUE THREAD

**Issue by xkenx1995** [debridmediamanager/zurg-testing#68]
**Expected Behavior**
Describe what you expect to happen.
partial refresh after media is added with DMM

**Actual Behavior**

2024-01-31T21:42:45.699-0800	INFO	manager	Detected changes! Refreshing 82 torrents
2024-01-31T21:42:46.447-0800	DEBUG	realdebrid	Got info for torrent 6SSQZYS6EAEJ4 (progress=100%)
2024-01-31T21:42:46.449-0800	DEBUG	manager	Saved torrent 6SSQZYS6EAEJ4 to file
2024-01-31T21:42:46.450-0800	INFO	manager	Fetched info for 82 torrents
2024-01-31T21:42:46.450-0800	INFO	manager	Compiled into 82 torrents, 0 were missing info
2024-01-31T21:42:46.450-0800	INFO	manager	Finished refreshing torrents
2024-01-31T21:42:48.064-0800	DEBUG	manager	Output of hook on_library_update:
Detected update on: 
Absolute path: Z:/
Detected update on: zurg
Absolute path: Z:/zurg
Detected update on: movies/Justice.League.2017.1080p.BluRay.REMUX.AVC.DTS-HD.MA.TrueHD.7.1.Atmos-FGT
Absolute path: Z:/movies/Justice.League.2017.1080p.BluRay.REMUX.AVC.DTS-HD.MA.TrueHD.7.1.Atmos-FGT
Detected update on: __all__/Justice.League.2017.1080p.BluRay.REMUX.AVC.DTS-HD.MA.TrueHD.7.1.Atmos-FGT
Absolute path: Z:/__all__/Justice.League.2017.1080p.BluRay.REMUX.AVC.DTS-HD.MA.TrueHD.7.1.Atmos-FGT
All updated sections refreshed

snippet of log, however plex does not trigger an update.

**Detailed Steps**
Detail out everything that you did leading up to the issue.

updated to hotfix 6. it was working fine on hotfix 4, but now I cant even get that one to work when I revert everything back.

**Operating System**
What OS are you running? (e.g., Windows, macOS, Linux)
If on Windows, are you using Windows Subsystem for Linux (WSL)?

Windows 11

**Environment Setup**
Are you using Docker, or are you using Zurg's binary?
If Docker, please include your `docker-compose.yml`.
If Zurg's binary, specify the version by running `./zurg version`

**Logs**
Upload your logs by going to http://ZURGIP:9999/logs/upload and paste the link here

https://0x0.st/HDpM.log

If you are running it in Docker, your container logs will also help.

**Rclone Configuration**
Attach or detail your `rclone.conf`.

mount zurg: z: --dir-cache 10s

**Zurg Configuration**
Include your config.yaml or any other relevant configuration files.

# Zurg configuration version
zurg: v1
token:  # https://real-debrid.com/apitoken

# do not change this if you are running it inside a docker container
host: "[::]"
port: 9999

# you can protect your zurg server with username+password auth
# username: yowmamasita
# password: 1234

# You can proxy all zurg requests like this:
# proxy: "http://[username:password@]host:port"
# proxy: "https://[username:password@]host:port"
# proxy: "socks5://[username:password@]host:port"

# How many requests in parallel should we send to Real-Debrid API?
concurrent_workers: 20

# How often should we check Real-Debrid API for file changes?
check_for_changes_every_secs: 15

# if true, saved rename info (if it exists) will be ignored
ignore_renames: false

# if true, it will strictly follow Real-Debrid API filename property
# setting to true should make zurg more compatible with rdt-client
retain_rd_torrent_name: false

# note: this is for cosmetic purposes only
# if true, zurg won't drop file extensions from directories
retain_folder_name_extension: false

# if true, zurg will delete Real-Debrid rar'ed torrents
# they are always compressed in a rar archive no matter what files you select
auto_delete_rar_torrents: false

# if true, during zurg initialization, it will fetch all downloads to unrestrict links faster
# it will also mount your download links in a special directory
use_download_cache: true

# BEWARE! THERE CAN ONLY BE 1 INSTANCE OF ZURG THAT SHOULD REPAIR YOUR TORRENTS
enable_repair: true

# this is useful for ensuring Plex adds your new content immediately
# uncomment the next line for triggering a partial scan
# on_library_update: sh plex_update.sh "$@"
# if on Windows uncomment the line below but comment out the one after it
on_library_update: '& .\plex_update.ps1 $args'

# buffer size when zurg is streaming files
network_buffer_size: 1048576 # 1 MiB

# true = send link to rclone and rclone will stream the file
# false = zurg will stream the file
serve_from_rclone: false

# true = zurg will check if the link is really working
# only relevant if serve_from_rclone is set to true
# as it already does this all the time if serve_from_rclone is false
verify_download_link: true

# force connect to real-debrid ipv6 addresses
# useful if you are blocked
force_ipv6: false

# sleep time after getting a 429 from Real-Debrid API
rate_limit_sleep_secs: 6

# time to wait before timing out
realdebrid_timeout_secs: 60

# api response failures until considered failed
retries_until_failed: 5

# use the fastest hosts from your location
# Run ./zurg network-test
# preferred_hosts:
#   - 20.download.real-debrid.com
#   - 21.download.real-debrid.com
#   - 22.download.real-debrid.com
#   - 23.download.real-debrid.com
#   - 30.download.real-debrid.com
#   - 31.download.real-debrid.com
#   - 32.download.real-debrid.com
#   - 34.download.real-debrid.com
#   - 40.download.real-debrid.com

# List of directory definitions and their filtering rules
directories:
  # Configuration for anime shows
  anime:
    group: media # directories on different groups have duplicates of the same torrent
    group_order: 10 # group order = priority, it defines who eats first on a group
    filters:
      - and: # you can use nested 'and' & 'or' conditions
        - has_episodes: true # intelligent detection of episode files inside a torrent
        - any_file_inside_regex: /^\[/ # usually anime starts with [ e.g. [SubsPlease]
        - any_file_inside_not_regex: /s\d\de\d\d/i # and usually anime doesn't use SxxExx

  shows:
    group: media
    group_order: 20
    filters:
      - has_episodes: true  # intelligent detection of episode files inside a torrent

  movies:
    group: media  # because anime, shows and movies are in the same group,
    group_order: 30 # and anime and shows has a lower group_order number than movies, all torrents that doesn't fall into the previous 2 will fall into movies
    only_show_the_biggest_file: true # let's not show the other files besides the movie itself
    filters:
      - regex: /.*/ # you cannot leave a directory without filters because it will not have any torrents in it

**Screenshots**

**Comment by yowmamasita** [comment]
maybe your plex token expired?

**Comment by xkenx1995** [comment]
> maybe your plex token expired?

oh didnt know thats possible? I check xml and it is still the same token

**Comment by yowmamasita** [comment]
the plex_update.ps1 issue is a community supported script, not really zurg. I suppose better to ask in the subreddit or in discord

**Comment by xkenx1995** [comment]
> the plex_update.ps1 issue is a community supported script, not really zurg. I suppose better to ask in the subreddit or in discord

gotcha. Any way to make it work through the config where it doesnt call on a script?

**Comment by yowmamasita** [comment]
Unfortunately the script is the only way to trigger plex
